### ISSUE TITLE
Adding more build servers into autotester infrastructure

### ISSUE THREAD

**Issue by iain-buclaw-sociomantic** [braddr/d-tester#64]
I had asked, and last time hardware was offered, it appeared to have been refused as there was no one willing to host it.

Is this still the case?

**Comment by braddr** [comment]
I don't believe I've ever turned down equipment that could do the job.  Though there's been a number of times I've turned down inadequate.  Generally speaking, must be available 24/7/365 -- the test equipment never stops building, essentially.  It must be big enough to actually complete builds in a timely manner (under an hour as a very rough guideline).  Etc.

And these days, yes, I generally really prefer not to physically host it and if possibly not be responsible for maintaining the underlying os and relevant security issues, etc.

**Comment by iain-buclaw-sociomantic** [comment]
Thanks, I could get the hardware spec of what we have so far this year.  Currently, all hardware being thrown out will have a few years on them, however am anticipating a refresh of equipment that may include rather recent boxes also.

Automation should solve the maintenance burden. :-)

**Comment by braddr** [comment]
The maintenance part is the least important part.. 'cause you're right, it's not that big a hassle.  Very very little of the equipment in the current fleet is terribly modern hardware.  Some of it is downright ancient an well past the point of needing to be sent to a recycle shop.  The only thing I'd be concerned about on old equipment is the health of the hdd/sdd drives.  If those are on the edge of failing, that could be a problem, but as long as the equipment is accessible, it can also easily just be left as a future problem.
